src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DayRent = 40;
```

Comment:
Consider renaming `DayRent` to a more descriptive name that follows the JavaScript naming conventions, such as `dailyRentCost`.